DSE_5.1_Release_Notes.md

```diff
@@ -2,6 +2,42 @@
 DSE 5.1.x is compatible with Apache Cassandra&trade; 3.11 and adds additional production-certified changes, if any. Components that are indicated with an asterisk (&ast;) (if any) are known to be updated since the prior patch version.
 
 
+# Release notes for 5.1.32
+18 July 2022
+
+## Components versions for DSE 5.1.32
+ * Apache Solr™ 6.0.1.0.2939&ast;
+ * Apache Spark™ 2.0.2.43&ast;
+ * Apache TinkerPop™ 3.2.11-20210716-faea8d16
+ * Apache Tomcat® 8.5.75
+ * DSE Java Driver 1.8.3-dse+20201217
+ * Netty 4.0.54.1.dse
+ * Spark JobServer 0.6.2.241&ast;
+
+## 5.1.32 DSE Core
+* Implemented ability to replace the THREE consistency level with an ALL_BUT_ONE consistency level. (DSP-22366)
+* Replaced `1.9.13` version of `jackson-core-asl` and `jackson-mapper-asl` to be `1.9.13.1.dse` which contains the security fix. (DSP-22389, [CVE-2019-10172](https://nvd.nist.gov/vuln/detail/CVE-2019-10172))
+* Removed dependency to Xerces2 Java XML Parser library. (DSP-22391, [CVE-2012-0881](https://nvd.nist.gov/vuln/detail/CVE-2012-0881), [CVE-2013-4002](https://nvd.nist.gov/vuln/detail/CVE-2013-4002))
+* Removed the PMD plugin from gradle config. (DSP-22575, [CVE-2019-7722](https://nvd.nist.gov/vuln/detail/CVE-2019-7722))
+* Fixed the cqlsh tool to restore the ability to use the EXECUTE AS command. (DSP-22417)
+* Fixed bug with overriding customer limits.d file if there are new values in the package upgrade. Complementary to DSP-21928. (DSP-22594)
+
+## 5.1.32 DSE Cassandra
+* Updated outdated `nofile` parameter value to `1048576` in `/etc/security/limits.d/cassandra.conf`. (DSP-21947)
+* Ported CASSANDRA-16987 and fixed python version check bug in cqlsh. (DSP-22517)
+
+## 5.1.32 DSE Search
+* Upgraded solr version which uses upgraded metadata-extractor 2.18.0 and XmpCore 6.1.11 libraries. (DSP-22406, [CVE-2019-14262](https://nvd.nist.gov/vuln/detail/CVE-2019-14262))
```

Comment:
`Upgraded solr version...` indicates we have bumped the solr version itself (at least to me), wouldn't:
`Upgraded libraries used in current solr version: metadata-extractor to 2.18.0 and XmpCore to 6.1.11` be a little bit more accurate?